src/test/java/org/broadinstitute/consent/http/service/DatasetServiceTest.java

```diff
@@ -79,10 +80,14 @@ class DatasetServiceTest {
   private DatasetServiceDAO datasetServiceDAO;
   @Mock
   private UserDAO userDAO;
+  @Mock
+  private User user;
+  @Mock
+  private Response response;
 
   private void initService() {
```

Comment:
If this was a `BeforeEach` method you wouldn't need to call it from every test method
```suggestion
  @BeforeEach
  public void initService() {
```